packages/devtools_app/macos/Runner.xcodeproj/project.pbxproj

```diff
@@ -215,7 +205,6 @@
 			isa = PBXNativeTarget;
 			buildConfigurationList = 331C80DE294CF71000263BE5 /* Build configuration list for PBXNativeTarget "RunnerTests" */;
 			buildPhases = (
-				2B0491DF3BA97DDCC97A774C /* [CP] Check Pods Manifest.lock */,
```

Comment:
Are these pbxproj changes intentional?

Reply:
Kind of - this file changes rather frequently as a result of running on `macos` (see https://github.com/flutter/devtools/commits/b398f1585348a921699dde42387d4459353ec3f3/packages/devtools_app/macos/Runner.xcodeproj/project.pbxproj) . I believe there is a reason we don't gitignore it but I forget why - @kenzieschmoll ?